sphinx/ext/intersphinx.py

```diff
@@ -340,21 +341,34 @@ def setup(app):
     return {'version': sphinx.__display_version__, 'parallel_read_safe': True}
 
 
-if __name__ == '__main__':
-    # debug functionality to print out an inventory
-    import sys
+def debug(argv):
```

Comment:
`mypy` expect to annotate here.
Please add `# type: (List[unicode]) -> None` here.
Then it might go well.